### ISSUE TITLE
Version bump breaks pip install django-appconf

### ISSUE THREAD

**Issue by pconerly** [jezdez/django-appconf#16]
hey @jezdez, 

I think that your version bump: https://github.com/jezdez/django-appconf/commit/a3e8adff68687c7f7b3933203f88bed688c093e3 has broken django-appconf installs.  I made a new virtualenv and did a `pip install django-appconf`, and what I got was:

```
(melvin)[peterconerly@melvinsrevenge]$ pip install --ignore-installed django-appconf==0.6
Downloading/unpacking django-appconf==0.6
  Could not find a version that satisfies the requirement django-appconf==0.6 (from versions: 1.0.1, 1.0.1)
Cleaning up...
No distributions matching the version for django-appconf==0.6
Storing debug log for failure in /Users/peterconerly/.pip/pip.log
```

I would upgrade to `1.0.1`, but `pinax-theme-bootstrap` requires `==0.6`.

I think you just need to do some pypi massaging.  Pip looks to pypi for 0.6, gets it, but then thinks that the version is 1.0.1.

No urgency!  I've got a good version in another project from yesterday and I'll do a `pip install -e` or something.  Enjoy your sunday.

**Comment by thijstriemstra** [comment]
Same here! Also opened a PR for the django-appconf dependency: https://github.com/pinax/pinax-theme-bootstrap/pull/91/

**Comment by senoadiw** [comment]
+1. in my case v0.5 gone from pypi breaks the GeoNode install script. Please reup and leave old versions alone for archival purposes. Thanks!

**Comment by bendemboski** [comment]
+1

**Comment by jezdez** [comment]
No commit has broken the packages on PyPI, but my stupidity, a horrible UX on PyPI when it comes to deleting releases and the whole package, see details here https://twitter.com/jezdez/status/566952554274500608 I'm also not the only one, https://github.com/getpelican/pelican/issues/1522

I'm waiting on the PyPI admins to re-enable me to upload the old versions. Sorry, y'all.

**Comment by andreagrandi** [comment]
@jezdez I was having the same problem and found this Issue. Could you please notify us when the problem is resolved? Thank you so much!

**Comment by andreagrandi** [comment]
p.s: for everyone, if you want to install the 0.6 version, you can change your line in requirements.txt with this:
```
git+git://github.com/jezdez/django-appconf.git#egg=0c9835f95b44db1b3eb1a9409f95a3ecd63b8ff5
```

**Comment by tzenderman** [comment]
Hey @jezdez any update on this? Thanks!

**Comment by jezdez** [comment]
Hi all, good news, I just uploaded 0.4, 0.4.1, 0.5, 0.6 and 1.0. :boom:

Thanks again for your patience.

As to the major UX issue with deleting release files that was easy to confuse with the ability to delete the whole project, there was some process there as well (thanks @r1chardj0n3s!):

- https://bitbucket.org/pypa/pypi/pull-request/68
- https://bitbucket.org/pypa/pypi/pull-request/69

I also opened https://bitbucket.org/pypa/pypi/pull-request/70 and will see how that goes.
